packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -81,6 +81,7 @@ export function PortableTextInput(props: PortableTextInputProps): ReactNode {
     editorRef: editorRefProp,
     elementProps,
     hotkeys,
+    initialFullscreen: initialFullscreenProp,
```

Comment:
i don't think u have to alias this one now 😄

Reply:
Third time's a charm?